web/angular-wallet/src/app/login/login.component.ts

```diff
@@ -3,7 +3,7 @@ import { Component, OnInit } from '@angular/core';
 // import { CreateService } from '../dashboard/setup/account/create.service';
 // import { NotifierService } from 'angular-notifier';
 import { Router } from '@angular/router';
-
+import {environment} from '../../environments/environment';
 @Component({
```

Comment:
We could `import { version } from '../../package.json'` and just use `npm version`. ALl we have to do is add `resolveJsonModule` to our `.tsconfig`, see https://www.typescriptlang.org/docs/handbook/compiler-options.html

Reply:
well, importing package.json into the code base is something I wouldn't do. Would work for an npm package or nodejs app, but not for a web app